### ISSUE TITLE
im on windows, what i am supposed to copy and paste?

### ISSUE THREAD

**Issue by UnknownnNOTAvailable** [rxzyx/prodigy-hack#130]
**what am i copying and pasting??**

**Comment by Slept66** [comment]
> i am on prodigy but what do i do now? ![Screenshot 2024-01-09 110249](https://private-user-images.githubusercontent.com/155848447/295263665-8c314508-3773-4b23-b98e-7c903396f583.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MDcyNzE3NDQsIm5iZiI6MTcwNzI3MTQ0NCwicGF0aCI6Ii8xNTU4NDg0NDcvMjk1MjYzNjY1LThjMzE0NTA4LTM3NzMtNGIyMy1iOThlLTdjOTAzMzk2ZjU4My5wbmc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjQwMjA3JTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI0MDIwN1QwMjA0MDRaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT0zNTFkNGI2OGU0NmRiYTRkNmM4MzAwZDI1NzU5MzkwMWFkNjI4MGQ2M2RiMDdlY2FiMThiZmZmMGM0MzA3ZmQzJlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCZhY3Rvcl9pZD0wJmtleV9pZD0wJnJlcG9faWQ9MCJ9.JMLLCC3Wrk82Bn2sqemTzBw8SsG12KUhChS6STAoo6o)

[Developers Tool (Where to Find) (Sleept66).docx](https://github.com/rxzyx/prodigy-hack/files/14188127/Developers.Tool.Where.to.Find.Sleept66.docx)
Good luck :D

**Comment by Slept66** [comment]
Copy and paste and follow the rest of the instructions :)

**Comment by hillychikon** [comment]
Is there a different way to do it, that's not working

**Comment by Slept66** [comment]
Not really, maybe try a different device, because as far as I know this is the only way to do it.

**Comment by hillychikon** [comment]
Okay, thanks

**Comment by Cheddeder** [comment]
wdym tf
